src/Parser/VengeanceDemonHunter/Modules/Features/AlwaysBeCasting.js

```diff
@@ -25,6 +32,34 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
     SPELLS.SOUL_BARRIER_TALENT.id,
     SPELLS.SPIRIT_BOMB_TALENT.id,
   ];
+
+  suggestions(when) {
+
+    const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
+
+    when(deadTimePercentage).isGreaterThan(0.3)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest('Your dead GCD time can be improved. Try to Always Be Casting (ABC); try to reduce the delay between casting spells and when you\'re not healing try to contribute some damage.')
+          .icon('spell_mage_altertime')
+          .actual(`${formatPercentage(deadTimePercentage)}% dead GCD time`)
```

Comment:
Using the `actual` var value is recommended so you don't need to keep this in sync with the value of `when()`